bsoid_app/clustering.py

```diff
@@ -4,103 +4,87 @@
 import joblib
 import numpy as np
 import streamlit as st
-import randfacts
 
 from bsoid_app.config import *
 from bsoid_app.bsoid_utilities import visuals
 from bsoid_app.bsoid_utilities.load_workspace import load_clusters
 from streamlit import caching
 
+min_cluster_range = float(os.environ.get('min_cluster_range', ''))
+max_cluster_range = float(os.environ.get('max_cluster_range', ''))
```

Comment:
does this mean these don't have defaults, if you're using an empty string at the end? or what is that syntax?

Reply:
tried to go in and add defaults for min and max cluster and for autosaving clustering